src/examples/deepExamples.cpp

```diff
@@ -118,20 +118,18 @@ readDeepScanlineFile (
     }
 }
 
-unsigned int getPixelSampleCount (int i, int j)
```

Comment:
revert this whitespace-only change

Reply:
done

Reply:
Nevermind, I figured out what's going on.